tools/tpm2_tool.h

```diff
@@ -59,7 +59,9 @@ bool tpm2_tool_onstart(tpm2_options **opts) __attribute__((weak));
  * @param flags
  *  Flags that tools may wish to respect.
  * @return
- *  0 on success.
+ *  0 on success
+ *  1 on failure
+ * -1 to show usage
```

Comment:
I think this is the standard, should we document this in the mapage? You can do this as a follow up to the patch series.

Reply:
As implemented the user doesn't see this return code, we set rc = 0 after printing usage in 2d80bdf.